lectia-3.js

```diff
@@ -1,2 +1,14 @@
-// https://miorita.scoalaweb.ro/lessons/3-while
-// Scrie un program care să o ajute pe Miorița să ajungă la pășunea din centrul spiralei
+reset();
+
+let lista = [6, 6, 6, 5, 5, 4, 4, 3, 3, 2, 2, 1, 1]
```

Comment:
La curs, într-adevăr am folosit array și for, ar sunt și alte metode de a rezolva. 

Dacă îți vine vreo idee, ar fi interesant de văzut la ce te gândești.